app/presenters/tasks/more_information.rb

```diff
@@ -5,7 +5,7 @@ def path
     end
 
     def not_applicable?
-      !FeatureFlags.more_information.enabled?
+      false
     end
```

Comment:
Can this method be deleted entirely if it is always false?

Reply:
not_applicable detaults to true in BaseTask, so needs to be implemented and overridden here